app/src/main/java/de/westnordost/streetcomplete/screens/settings/debug/ShowLinksActivity.kt

```diff
@@ -1,33 +0,0 @@
-package de.westnordost.streetcomplete.screens.settings.debug
-
-import android.os.Bundle
-import androidx.recyclerview.widget.DividerItemDecoration
-import androidx.recyclerview.widget.LinearLayoutManager
-import de.westnordost.streetcomplete.R
-import de.westnordost.streetcomplete.databinding.FragmentShowLinksBinding
-import de.westnordost.streetcomplete.screens.BaseActivity
-import de.westnordost.streetcomplete.screens.user.links.GroupedLinksAdapter
-import de.westnordost.streetcomplete.util.ktx.openUri
-import de.westnordost.streetcomplete.util.viewBinding
-import org.koin.androidx.viewmodel.ext.android.viewModel
-
-/** activity only used in debug, to show all achievement links */
-class ShowLinksActivity : BaseActivity() {
```

Comment:
@FloEdelmann I removed this because it is more convenient to preview the links directly in the IDE, see LazyLinksColumn